daemon/src/common/Color.cpp

```diff
@@ -1,7 +1,7 @@
 /*
 ===========================================================================
 Daemon BSD Source Code
-Copyright (c) 2015, Daemon Developers
+Copyright (c) 2013-2016, Daemon Developers
 All rights reserved.
```

Comment:
I think it is valid to say that the "Daemon BSD source code" (which this file is part of) started in 2013 and it makes it easier to update all the copyrights in the future.